### DISCUSSION TITLE
196

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#173] [General]
# 潮流周刊第196期 - 新显示器

封面图拍摄于最近书房新置办的戴尔 U2723QE 显示器，简单配置了一下，效果挺好的，幸福感增加了不少。

https://weekly.tw93.fun/posts/196-%E6%96%B0%E6%98%BE%E7%A4%BA%E5%99%A8

**Comment by Yekaifei** [comment]
你好，我想问问用什么搭建的这个网站呢

**Reply by tw93** [reply]
可以见 github https://github.com/tw93/weekly
此外你也可以fork一个自己使用

**Comment by gushisan** [comment]
快乐的第 4 点「思维限制我们最求快乐」有错别字「追求」

**Reply by tw93** [reply]
谢谢指出

**Comment by hnynf** [comment]
用的是什么工具打开的 ppt 呢？

**Reply by tw93** [reply]
arc 浏览器而已

**Comment by fundgao** [comment]
显示器，我买的是华为34寸带鱼屏，这可等于2.5个普通屏幕，幸福感满满

**Comment by GetToSet** [comment]
同款显示器，我的是带 HDR 的 U2723QX，这款和 Apple 设备的兼容性特别好，而且能自己更新固件。

**Comment by Nymph2333** [comment]
我用的红米27寸4K，色彩还原度很好，价格还不到2000，不仅接口很多，和Mac完全兼容
